airflow/providers/singularity/example_dags/example_singularity.py

```diff
@@ -23,19 +23,17 @@
 from airflow.providers.singularity.operators.singularity import SingularityOperator
 from airflow.utils.dates import days_ago
 
-default_args = {
-    'owner': 'airflow',
-    'depends_on_past': False,
-    'email': ['airflow@example.com'],
-    'email_on_failure': False,
-    'email_on_retry': False,
-    'retries': 1,
-    'retry_delay': timedelta(minutes=5),
-}
-
 with DAG(
     'singularity_sample',
-    default_args=default_args,
+    default_args={
+        'owner': 'airflow',
+        'depends_on_past': False,
+        'email': ['airflow@example.com'],
+        'email_on_failure': False,
+        'email_on_retry': False,
```

Comment:
```suggestion
```

Not relevant to the example, so lets remove it :)

Reply:
Removed other default_arg args